### ISSUE TITLE
Tab moves focus needs ux love

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#4742]
Steps to Reproduce:

1. command palette > Toggle use of tab key for setting focus
2. notice red notification in the status bar - I believe we can find a better color for this

Though there should be some contrast since this is an important scenario for accessibility

![screen shot 2016-03-29 at 15 39 27](https://cloud.githubusercontent.com/assets/1926584/14109918/a2f53310-f5c4-11e5-94d7-b9ce9ddb4674.png)

**Comment by bgashler1** [comment]
Duplicate #2503
